### PR TITLE
Restaurants Component

### PR DESCRIPTION
**Pull request by robin-hodge**

### CODE REVIEW

File: .gitignore
Comment (west-country) [inline .gitignore]: possible conflict with our approach to scss

File: package.json
Comment (west-country) [inline package.json]: Not using react-bootstrap, Rayna advised against

File: src/App/App.js
Comment (west-country) [inline src/App/App.js]: indent

File: src/Main/RestaurantsContainer/RestaurantsContainer.js
Comment (west-country) [inline src/Main/RestaurantsContainer/RestaurantsContainer.js]: don't need parentheses around response.json

File: src/Main/RestaurantsContainer/RestaurantsContainer.js
Comment (west-country) [inline src/Main/RestaurantsContainer/RestaurantsContainer.js]: No error handling - mentioned in API readme, but not sure if necessary?

File: src/Main/RestaurantsContainer/RestaurantsContainer.js
Comment (west-country) [inline src/Main/RestaurantsContainer/RestaurantsContainer.js]: please explain data-restaurant-id in comment for team who will use it later or delete - right now not doing anything

File: src/Main/RestaurantsContainer/RestaurantsContainer.scss
Comment (west-country) [inline src/Main/RestaurantsContainer/RestaurantsContainer.scss]: missing space. colour didn't seem to match for us, might need changing

**Review by west-country — COMMENTED** [review]
nice job, just few minor things

File: .gitignore
Comment (robin-hodge) [reply .gitignore]: Done!

File: package.json
Comment (robin-hodge) [reply package.json]: Done

File: src/App/App.js
Comment (robin-hodge) [reply src/App/App.js]: Done!

File: src/Main/RestaurantsContainer/RestaurantsContainer.js
Comment (robin-hodge) [reply src/Main/RestaurantsContainer/RestaurantsContainer.js]: Done!

File: src/Main/RestaurantsContainer/RestaurantsContainer.scss
Comment (robin-hodge) [reply src/Main/RestaurantsContainer/RestaurantsContainer.scss]: We will use the variable you created on merge?

File: src/Main/RestaurantsContainer/RestaurantsContainer.js
Comment (robin-hodge) [reply src/Main/RestaurantsContainer/RestaurantsContainer.js]: Resolved

File: .gitignore
Comment (tauroid) [inline .gitignore]: we'll figure this out later idk

File: src/Main/RestaurantsContainer/RestaurantsContainer.js
Comment (tauroid) [inline src/Main/RestaurantsContainer/RestaurantsContainer.js]: I thought you were gonna do response.ok etc but won't push it if you don't want to (was Rayna talking about Loading... rather than error handling?)

**Comment by tauroid** [comment]
Looks good, will approve once we've merged our tasks

File: src/Main/RestaurantsContainer/RestaurantsContainer.js
Comment (robin-hodge) [reply src/Main/RestaurantsContainer/RestaurantsContainer.js]: All good as discussed

**Review by west-country — APPROVED** [review]

**Review by tauroid — CHANGES_REQUESTED** [review]
Move RestaurantContainer/ out of Main/ as discussed please thanks

**Review by tauroid — APPROVED** [review]

**Pull request merged**
